### ISSUE TITLE
Egress support: update `bpf policy add` subcommands to support egress updates

### ISSUE THREAD

**Issue by ianvernon** [cilium/cilium#3114]
Add a flag which indicates whether we are updating ingress or egress policy.

**Comment by ianvernon** [comment]
Work to do this has been merged:

```
vagrant@runtime1:~/go/src/github.com/cilium/cilium$ sudo cilium bpf policy add 29898 ingress 12345 80/TCP
$ sudo cilium bpf policy get 29898
DIRECTION   LABELS (source:key[=value])   PORT/PROTO   BYTES   PACKETS   
Ingress     12345                         80/TCP       0       0    
```

**Comment by ianvernon** [comment]
Addressed by: 18fbfad4756d8336875aa4ff31c94d3903254aa3 .
